src/SmartComponents/HybridInventoryTabs/helpers.js

```diff
@@ -88,7 +93,8 @@ export const useGetEntities =
       );
 
       edgeData = devicesData?.data?.devices || [];
-      enforceEdgeGroups = devicesData?.data?.enforce_edge_groups;
+      enforceEdgeGroups =
+        devicesData?.data?.enforce_edge_groups || !isInventoryGroupsEnabled;
```

Comment:
What is the reason to negate the isInventoryGroupsEnabled value? From the semantics of this code snippet, I think enforceEdgeGroups must be true if `devicesData?.data?.enforce_edge_groups` evaluates to false/undefined and `isInventoryGroupsEnabled` is true:
```suggestion
        devicesData?.data?.enforce_edge_groups || isInventoryGroupsEnabled;
```

Reply:
> What is the reason to negate the isInventoryGroupsEnabled value? From the semantics of this code snippet, I think enforceEdgeGroups must be true if `devicesData?.data?.enforce_edge_groups` evaluates to false/undefined and `isInventoryGroupsEnabled` is true:

@gkarat thank you for having a look. It is because of this condition, in the acceptance criteria ^:

1. When “enforce_edge_groups” is false or missing 
2. When unleash feature flag “edgeParity.inventory-groups-enabled” is false
3. display edge group

Let me know if this makes sense.